src/test/java/utils/TestContext.java

```diff
@@ -0,0 +1,12 @@
+package utils;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class TestContext {
+    public WebDriver driver;
+
+    public TestContext() {
+        this.driver = new ChromeDriver();
```

Comment:
Driver should not be created in the context. Should be created outside and saved to the context.